functions-and-operators/numeric-functions-and-operators.md

```diff
@@ -24,33 +24,37 @@ TiDB supports all of the [numeric functions and operators](https://dev.mysql.com
 
 | Name                                                                                                      | Description                                                       |
 |:----------------------------------------------------------------------------------------------------------|:------------------------------------------------------------------|
```

Comment:
Tip for reviewers: 
The following changes only involve sorting the functions in this table.